sklego/__init__.py

```diff
@@ -1 +1,9 @@
-__version__ = "0.7.0"
+import sys
+
+if sys.version_info >= (3, 8):
+    from importlib import metadata
+else:
+    import importlib_metadata as metadata
+
+__title__ = "sklego"
+__version__ = metadata.version("scikit-lego")
```

Comment:
This took me sooo long to debug due to the different naming between distribution and module

Reply:
Yeah, that's certainly a bit awkward.